### DISCUSSION TITLE
Vite v4 is out, Pocketbase UI is using v3

### DISCUSSION THREAD

**Discussion by dschissler** [pocketbase/pocketbase#1544] [Q&A]
@ganigeorgiev Since you just released a new version then now would be a good time to upgrade Vite and the Svelte plugin.  You shouldn't have to do anything at all for the upgrade except change the package version.  Especially since you made such a simple vite config.

**Comment by ganigeorgiev** [comment]
It will be updated together with the planned UI improvements in v0.12.
